libs/ui/editor/src/lib/Blocks/types.ts

```diff
@@ -123,56 +123,56 @@ export enum EmbedType {
   Other = 'other'
 }
 
-export interface FacebookPostEmbed {
+interface FacebookPostEmbed {
```

Comment:
Shouldn't these be exported?

Reply:
same as with `export type BlockMap = Record<string, BlockCaseProps>`